### PR TITLE
Add OpenAiImageClient auto-configuration

### PR DESCRIPTION
**Pull request by ThomasVitale**
Fixes gh-289

### CODE REVIEW

File: spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/openai/OpenAiImageProperties.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+@ConfigurationProperties(OpenAiImageProperties.CONFIG_PREFIX)
+public class OpenAiImageProperties extends OpenAiParentProperties {
+
+	public static final String CONFIG_PREFIX = "spring.ai.openai.image";
+
+	/**
+	 * Options for OpenAI Image API.
+	 */
+	@NestedConfigurationProperty
+	private OpenAiImageOptions options = OpenAiImageOptions.builder().build();
```
Comment (ThomasVitale) [inline spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/openai/OpenAiImageProperties.java:38]: In the `OpenAiChatProperties` class, a default model is specified there.
Here, I didn't specify one as the default is already included in the `OpenAiImageOptions` class (unlike the `OpenAiChatOptions` class which doesn't have a default model).

**Comment by tzolov** [comment]
Thank you @ThomasVitale , will review it shortly

**Comment by tzolov** [comment]
Great  job @ThomasVitale ! 
Rebased and merged at: 41a459a10b42ae891dbeb1242cadff7ee3ea6bb2

**Comment by ThomasVitale** [comment]
@tzolov thank you!

**Pull request closed without merging**